documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,188 @@
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-1.png "Jenkins New Item")
+   </p>  
+
+1. For **Definition** in the **Pipeline** options, choose **Pipeline script from SCM**.
+
+1. For **SCM**, choose **Git**.
+
+1. For **Repository URL** in the **Repositories** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in the **Credentials** section.
+   <p align="center">
+   ![Create Pipeline Job](../images/JenkinsNewItemPipeline-2.png "Jenkins New Item")
+   </p>
+
+1. For **Branch Specifier** in the **Branches to build** section, enter the branch name `*/1_REST_persist_in_Memory`.
+
+1. Choose **Save**.
+
+1. To run your pipeline, choose **Build Now** in the job UI.
+
+
+## Add a Build Step
+
+1. In your `Jenkinsfile`, add the following code snippet after `stage("prepare") { ... }`:
+   ```
+   stage('build') {
+       mtaBuild script: this
+   }
+   ```
+   The `mtaBuild` step calls a build tool to build a multi-target application (MTA). The tool consumes an MTA descriptor that contains the metadata of all entities which comprise an application or are used by one during deployment or runtime, and the dependencies between them. For more information about MTAs, see [sap.com][sap].
+
+1. Create the MTA descriptor file with the name `mta.yaml` in the root level of the repository. Insert the following code:
+   ```
+   _schema-version: 2.1.0
+   ID: com.sap.piper.node.hello.world
+   version: 1.0.0
+   description: A Hello World sample application
+   provider: SAP Sample generator
+   modules:
+     - name: piper.node.hello.world
+       type: nodejs
+       path: .
+   ```
+
+1. Configure the step to build an MTA for the Cloud Foundry environment. Create the configuration file `.pipeline/config.yml` relative to the root level of the repository and insert the following content:
+   ```
+   general:
+   steps:
+     mtaBuild:
+       buildTarget: 'CF'
+   ```
+   For additional information about the configuration, have a look at the [Common Configuration Guide][resources-configuration] and the [MTA build step documentation][resources-step-mtabuild].
+
+1. Save your changes to your remote repository.
+
+1. To run your pipeline, choose **Build Now** in the job UI. 
+
+## Add a Deploy Step
+
+1.  In your `Jenkinsfile`, add the following code snippet:
+   ```
+   stage('deploy') {
+       def mtarFilePath = commonPipelineEnvironment.getMtarFilePath()
+       cloudFoundryDeploy( script: this, mtaPath: mtarFilePath)
+   }
+   ```
+   The `cloudFoundryDeploy`  step calls the Cloud Foundry command line client to deploy into SAP Cloud Platform.
+
+1. To configure the step to deploy into the Cloud Foundry environment, in your repository, open or create the `.pipeline/config.yml` and add the following content:
```

Comment:
> open or create

The file was added by the guided tour earlier. Why not simply 'open' instead of 'open or create'